sdk/storage/storage-queue/samples/typescript/proxyAuth.ts

```diff
@@ -0,0 +1,45 @@
+/* 
+ Setup: Enter your storage account name and shared key in main()
+*/
+
+import { SharedKeyCredential, QueueServiceClient } from "../../src"; // Change to "@azure/storage-blob" in your package
+
+async function main() {
+  // Enter your storage account name and shared key
+  const account = "";
+  const accountKey = "";
+
+  // Use SharedKeyCredential with storage account and account key
```

Comment:
also add this second line of comment

  // SharedKeyCredential is only avaiable in Node.js runtime, not in browsers